src/ios/CDVCapture.m

```diff
@@ -266,7 +267,18 @@ - (void)captureVideo:(CDVInvokedUrlCommand*)command
         // iOS 4.0
         if ([pickerController respondsToSelector:@selector(cameraCaptureMode)]) {
             pickerController.cameraCaptureMode = UIImagePickerControllerCameraCaptureModeVideo;
-            // pickerController.videoQuality = UIImagePickerControllerQualityTypeHigh;
+            switch((int)([quality doubleValue]*10)) {
```

Comment:
Is this operation always safe to do here and necessary, or shouldn't we have a more "direct" _switch_ on `quality` value itself?

For example if the user has bad options `{quality: null}`, or _undefined_, a string, a boolean, or any wrong/not number value for `quality` param. Or maybe the use of `NSNumber* quality = [options objectForKey:@"quality"];` will fail first in those cases?